tests/props/host/host.c

```diff
@@ -165,29 +177,46 @@ int main(int argc, const char* argv[])
         oe_put_err("oe_create_enclave(): result=%u", result);
 
     /* Check expected enclave property values */
-    if (is_signed)
+    switch (test_type)
     {
+    case 0: /* unsigned */
```

Comment:
Can you replace the case labels with the enum? That was part of the reason for the original suggestion to use enums.